app/assets/v2/js/pages/bounty_details.js

```diff
@@ -815,10 +846,12 @@ var do_actions = function(result) {
   let show_start_stop_work = is_still_on_happy_path && !should_block_from_starting_work && is_open && !isBountyOwner(result);
   let show_github_link = result['github_url'].substring(0, 4) == 'http';
   let show_submit_work = is_open && !has_fulfilled;
+  let show_notify_funder = is_open && has_fulfilled;
```

Comment:
@PixelantDesign Would you like the button to just not show when it has been too soon to be pushed again and should be disabled for the fulfiller?

Reply:
@PixelantDesign Please see question above!

Reply:
@pinkiebell Are you recommending to not show the button instead of graying it out?

show_notify_funder relies on has_fulfilled to make sure the user is the fulfiller. If they are, they will always get the notify funder button, and it only grays out (notify_funder_enabled == false) if the user already clicked it or the model says the user can't notify the funder anymore according to the rules set in the requirements. Could you expand on this one?

Reply:
@oogetyboogety 
You can leave it the way it is, I guess.
Unless @PixelantDesign has another opinion 😄